.github/workflows/conformance.yaml

```diff
@@ -586,8 +586,7 @@ jobs:
       - uses: ./.github/actions/run-tests
         with:
           k8s-version: ${{ matrix.k8s-version }}
-          kind-config: ./scripts/config/kind/vap-v1beta1.yaml
-          kyverno-configs: standard,generate-validating-admission-policy
+          kyverno-configs: standard
```

Comment:
Why did you remove the `generate-validating-admission-policy` config ?

Reply:
why is it required? This config sets the flag `--generateValidatingAdmissionPolicy` to true which isn't required for webhook configuration afaik.

Reply:
Ok

Reply:
Thanks for clearing it up